javalin/src/main/java/io/javalin/http/HttpCode.kt

```diff
@@ -63,13 +63,28 @@ enum class HttpCode(val status: Int, val message: String) {
     LOOP_DETECTED(508, "Loop Detected"),
     @Deprecated("Obsoleted by https://datatracker.ietf.org/doc/status-change-http-experiments-to-historic/")
     NOT_EXTENDED(510, "Not Extended"),
-    NETWORK_AUTHENTICATION_REQUIRED(511, "Network Authentication Required");
+    NETWORK_AUTHENTICATION_REQUIRED(511, "Network Authentication Required"),
+    CUSTOM_HTTP_CODE(599, "Non-standard HTTP code");
```

Comment:
Looks like `599` is often used to represent an http connection error (like e.g. timeout by proxies etc.). If we don't know status code, UNKNOWN(-1) could be a better choice to represent all unknown status codes.